CMakeLists.txt

```diff
@@ -144,20 +163,36 @@ add_executable (
   terminal/PortForwardClientListener.cpp
   terminal/SshSetupHandler.cpp
   )
-target_link_libraries (
-  et
-  LINK_PUBLIC
-  TerminalCommon
-  EternalTCP-static
-  ${CMAKE_THREAD_LIBS_INIT}
-  ${PROTOBUF_LIBRARIES}
-  ${GLOG_LIBRARIES}
-  ${GFLAGS_LIBRARIES}
-  ${sodium_LIBRARY_RELEASE}
-  ${UTEMPTER_LIBRARIES}
-  resolv
-  util
+IF(!FREEBSD)
```

Comment:
If you follow the change above, you can also replace this IF by using ${CORE_LIBRARIES}